cobalt/browser/application.cc

```diff
@@ -589,6 +589,28 @@ void AddCrashHandlerAnnotations() {
   }
 }
 
+void AddCrashHandlerApplicationState(base::ApplicationState state) {
+  auto crash_handler_extension =
+      static_cast<const CobaltExtensionCrashHandlerApi*>(
+          SbSystemGetExtension(kCobaltExtensionCrashHandlerName));
+  if (!crash_handler_extension) {
+    LOG(INFO) << "No crash handler extension, not sending application state.";
```

Comment:
Should these be `DLOG`s?

Reply:
They were kept as LOGs to be consistent with the crash handler annotations above, but DLOGs makes more sense given the frequency.